src/main/java/org/takes/rs/RsWithoutHeader.java

```diff
@@ -52,18 +53,16 @@ public RsWithoutHeader(final Response res, final CharSequence name) {
         super(
             new Response() {
                 @Override
+                @SuppressWarnings("unchecked")
                 public Iterable<String> head() throws IOException {
-                    final Collection<String> head = new LinkedList<String>();
                     final String prefix = String.format(
                         "%s:", name.toString().toLowerCase(Locale.ENGLISH)
                     );
-                    for (final String hdr : res.head()) {
-                        if (!hdr.toLowerCase(Locale.ENGLISH)
-                            .startsWith(prefix)) {
-                            head.add(hdr);
-                        }
-                    }
-                    return head;
+                    return new Concat<String>(
+                        res.head(),
+                        Collections.EMPTY_LIST,
```

Comment:
@krzyk, I tried using Collections.emptyList() but it actually returns List<Object>, which cannot cast back to Iterable<String> but raw type can do so.